packages/sync/src/conflicts/conflictLink.ts

```diff
@@ -40,7 +40,15 @@ export const conflictLink = (config: DataSyncConfig): ApolloLink => {
         }
       } else {
         // resolve on client
-        resolvedConflict = config.conflictStrategy(operation.operationName, data.serverState, data.clientState);
+        if (config.resolvers && config.resolvers[operation.operationName]) {
+          config.resolvers[operation.operationName](data.serverState, data.clientState)
+        }
+
+        // use global function, if present
+        if (config.conflictStrategy) {
```

Comment:
@wtrocki Do we have to keep this for compatibility with the previous code?

Reply:
No.. This is relatively low priority change but we need  to change API before it hits users.

Reply:
👍 ok, will remove that then